portal-web/test/functional/com/liferay/portalweb/macros/FormsAdminNavigator.macro

```diff
@@ -114,6 +114,17 @@ definition {
 			value1 = "Entries");
 	}
 
+	macro gotoEntriesLabel {
+		Click(
+			key_fieldName = "${key_value}",
+			locator1 = "FormFields#TEXT_FIELD_LABEL_REPORT_CARD");
+			
+		AssertVisible(
+			key_type = "${key_key_type}",
+			key_value = "${key_value}",
+			locator1 = "FormViewBuilder#ENTRY_ANY_LABEL");
```

Comment:
@lucasperj Can you please change the location of this path? The App Builder folder does not belong to Forms. Thanks!

Reply:
Also, is there a way to confirm the page has scrolled down instead of checking if the label is visible? Thanks!

Reply:
Done! I'm following the new workflow to resolve conflicts and changes, using Brian branch as reference to avoid conflicts. Thank you.

Reply:
@lucasperj This path went back to the App Builder folder. Can you please check this? Thanks!

Reply:
@carolmariaabb  Check!